django_freeradius/base/models.py

```diff
@@ -819,3 +820,36 @@ def _remove_files(self):
         path = getattr(self, strategy_filemap.get(self.strategy)).path
         if os.path.isfile(path):
             os.remove(path)
+
+
+class AbstractRadiusToken(AutoUsernameMixin, models.Model):
```

Comment:
Use this as a base: https://github.com/openwisp/openwisp-utils/blob/master/openwisp_utils/base.py#L8

and get rid of `created` here, it will be inherited automatically